common/dpc_common.hpp

```diff
@@ -0,0 +1,52 @@
+//==============================================================
```

Comment:
Why are we committing this file here? Isn't it supposed to be inside oneAPI package only?

Reply:
per request by Swapna, it is for reference only.